qtpy/tests/test_qtlocation.py

```diff
@@ -1,10 +1,15 @@
 import pytest
 from qtpy import PYQT5, PYSIDE2
 
+
 @pytest.mark.skipif(not (PYQT5 or PYSIDE2), reason="Only available in Qt5 bindings")
 def test_qtlocation():
     """Test the qtpy.QtLocation namespace"""
     from qtpy import QtLocation
+
+    if PYQT5:
```

Comment:
```suggestion
    if PYSIDE2:
```

Reply:
My bad. The other file had 400 instances of `not PYSIDE2` and I got turned around!

Reply:
Imagine that, the tests work now!